crates/nargo/src/cli/prove_cmd.rs

```diff
@@ -50,26 +71,38 @@ pub fn prove_with_path<P: AsRef<Path>>(
     // Since the public outputs are added onto the public inputs list, there can be duplicates.
     // We keep the duplicates for when one is encoding the return values into the Verifier.toml,
     // however we must remove these duplicates when creating a proof.
-    compiled_program.circuit.public_inputs =
-        dedup_public_input_indices(compiled_program.circuit.public_inputs);
+    let proof = {
```

Comment:
Nitpick, but lets avoid creating an unnecessary block here. It prevents these new variables from being used elsewhere but I don't think it adds enough to merit introducing this new style to the code base.